contracts/btc-finality/src/contract.rs

```diff
@@ -71,9 +71,12 @@ fn get_blocks_per_year(deps: &mut DepsMut) -> Result<u64, ContractError> {
             cosmwasm_std::Binary::new("".into()),
         )?;
         // Deserialize protobuf
-        let res_decoded = anybuf::Bufany::deserialize(&res).unwrap();
+        let res_decoded =
+            anybuf::Bufany::deserialize(&res).expect("Static value must not fail to deserialize");
         // See https://github.com/cosmos/cosmos-sdk/blob/8bfcf554275c1efbb42666cc8510d2da139b67fa/proto/cosmos/mint/v1beta1/query.proto#L35-L36
-        let res_params = res_decoded.message(1).unwrap();
+        let res_params = res_decoded
+            .message(1)
+            .expect("Static value must contain a message at index 1");
```

Comment:
Better. Another (better) option is to add a `ProtobufDecode` from `BufanyError` contract error, and return it here through `?`.